README.md

```
@@ -0,0 +1,73 @@
+</br>
+
+## 📌 프로젝트 기능 및 기술 적용
+### 📄 단기 프로모션 관리 및 대규모 트래픽 감당
+> 저희 서비스에서는 단기 프로모션을 진행하는 업체가 해당 서비스로 대행 판매를 요청하게 됩니다.
+> 이에 따라 프로모션의 기간, 내용, 업체, 프로모션에 해당되는 상품과 상품 재고 등을 관리하게 됩니다.
+> 유저는 프로모션의 기간 동안 정해진 할인 금액으로 상품을 장바구니에 담고, 주문 및 결제를 진행할 수 있습니다.
+> 결제가 완료되면 해당 주문이 완료되었다는 알림을 수신할 수 있습니다.
+![image](https://github.com/user-attachments/assets/48a896e6-87cc-4f15-8401-116f941929b7)
+
+
+### 🌱 유저 서비스의 JWT/Passport 도입
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)